data/tutorials/collections.yaml

```
@@ -37,3 +37,8 @@
   type: proficiency
   name: Learn Pulumi
   description: Intro-level walkthroughs to help you learn core Pulumi infrastructure-as-code concepts.
+
+- id: using-esc
+  type: proficiency
```

I was sort of imagining that this collection would be of type `product` and be labeled just `esc`, such that the URL would then become https://pulumi.com/tutorials/esc (or alternatively `pulumi-esc`, if we wanted to be a bit more formal about it). This way, other `product`-type tutorials could come up alongside it (`iac`, `insights`, `cloud`) and things would generally be consistent. Any opposition to that? (That's not to say we can't have more collections later of course; just thinking it'd be good to start with product-specific buckets being the main ones initially, assuming users will most likely want to pivot on those primarily.)

```suggestion
- id: pulumi-esc
  type: product
```

---

yeah, i mostly just cargo-culted the existing collection. I like the idea of making product-specific collections for each product and also including product-integration-style posts under a collection like "Pulumi as a Platform"  other topics as well as collections to cover "ingredients" like Crossguard/Deployments/etc.

What does changing the `type` do for the end result rendering?

---

>What does changing the type do for the end result rendering?

Nothing yet, but I'm thinking we might use that for grouping or coloring the tags somehow, etc., sometime down the road.